src/modules/Dashboard/features/Applet/DashboardAppletSettings/DashboardAppletSettings.utils.tsx

```diff
@@ -12,10 +12,11 @@ import {
   // ShareAppletSetting,
 } from 'shared/features/AppletSettings';
 import { Mixpanel, SettingParam, isManagerOrOwner } from 'shared/utils';
+import { Item as ItemNavigation } from 'shared/components/NavigationMenu/NavigationMenu.types';
```

Comment:
```suggestion
import { Item as ItemNavigation } from 'shared/components/NavigationMenu';
```
The type Item is exporting from the corresponding index.ts